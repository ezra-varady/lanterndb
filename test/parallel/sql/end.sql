SELECT COUNT(*) FROM sift_base10k;
SELECT * from sift_base10k WHERE id=4444;
