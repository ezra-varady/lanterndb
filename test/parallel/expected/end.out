BING BING WAHOO
SELECT COUNT(*) FROM small_world;
 count 
-------
    16
(1 row)

